src/transactions/OfferTests.cpp

```diff
@@ -1292,6 +1431,15 @@ TEST_CASE("create offer", "[tx][offers]")
                                           {xlm, usd, oneone, 110});
             });
         });
+        for_versions_from(10, *app, [&] {
+            auto offer = market.requireChangesWithOffer({}, [&] {
+                return market.addOffer(a1, {xlm, usd, oneone, 9});
```

Comment:
I think we should change the amounts to be the ones around the edge case:
so use "10" instead of "9", and use "11" instead of "111"

Reply:
Done.